<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNewTimer</class>
 <widget class="QDialog" name="DlgNewTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="flayoutParams">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelSound">
       <property name="text">
        <string>&amp;Sound:</string>
       </property>
       <property name="buddy">
        <cstring>comboSound</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboSound">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPeriod">
       <property name="text">
        <string>Pe&amp;riod:</string>
       </property>
       <property name="buddy">
        <cstring>spinPeriod</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinPeriod">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelShortcut">
       <property name="text">
        <string>Shortcut:</string>
       </property>
       <property name="buddy">
        <cstring>editShortcut</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QKeySequenceEdit" name="editShortcut"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelInvalidShortcut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string> Invalid shortcut, please enter another one </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspacerLayout">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayoutButtons">
     <item>
      <spacer name="hspacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboSound</tabstop>
  <tabstop>spinPeriod</tabstop>
  <tabstop>editShortcut</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
